T925716, T1053765: Update showEditorAlways (#3378)
DevExpress/devextreme-documentation: api-reference/_hidden/GridBaseColumn/showEditorAlways.md

---
---
##### shortDescription

> Specifies whether the column displays its values in editors in display state.

Reviewer: You must use an article with the word "state". Make sure to make this correction document wide. Also, rewrite sentence not to use "in" so often so close to each other.

---

> Column cells can be in display and edit states. In display state, a cell value is text. In edit state, the cell contains an editor that allows users to edit the cell value. To use editors in both states, set the **showEditorAlways** property to **true**.

Reviewer: can be? can be what? rewrite
a cell value is text? rewrite
In an edit state, the cell contains an editor that allows users to edit that cell's value.

> Column cells can be in display and edit states. In display state, a cell value is text. In edit state, the cell contains an editor with the value and allows users to edit it. To show cell values in editors in edit and display states, set the **showEditorAlways** property to **true**.

Reviewer 2: > In edit state, the cell contains an editor with the value and allows users to edit it.

"It" is ambiguous.

> To show cell values in editors in edit and display states

"In" is repeated.

```suggestion
Column cells can be in display and edit states. In display state, a cell value is text. In edit state, the cell contains an editor that allows users to edit the cell value. To use editors in both states, set the **showEditorAlways** property to **true**.
```

> In "cell" or "batch" edit mode, users can edit cells. For "row", "form", or "popup" edit modes, the component shows editors but discards the changes. In these edit modes, users need to click the "edit" button to edit values. 

Reviewer 2: It wasn't obvious to me that all this applies when showEditorAlways is enabled.

```suggestion
If [edit mode]() is *"cell"* or *"batch"*, users can change cell values in display state. In *"row"*, *"form"*, or *"popup"* edit modes, the component shows editors but discards the changes. In these edit modes, users need to click the Edit button to change cell values. 
```

> If [edit mode]() is *"cell"* or *"batch"*, users can change cell values in display state. In *"row"*, *"form"*, or *"popup"* edit modes, the component shows editors but discards the changes. In these edit modes, users need to click the Edit button to change cell values. 

Reviewer: ```suggestion
If [edit mode]() is *"cell"* or *"batch"*, users can change cell values in display state [this sentence doesn't make sense, because you are not clear about the relationship of cell values and the display state; clarify]. In *"row"*, *"form"*, or *"popup"* edit modes, the component displays editors, but discards the changes. In these edit modes, users need to click the Edit button to change cell values. 
```

[note]

This property has the following specifics:

> - The default value of this property depends on the column's [dataType](/api-reference/_hidden/GridBaseColumn/dataType.md '{basewidgetpath}/Configuration/columns/#dataType'). For boolean columns, it is **true**; for columns of other types - **false**. 

Reviewer: ```suggestion
- The default value of this property depends on the column's [dataType](/api-reference/_hidden/GridBaseColumn/dataType.md '{basewidgetpath}/Configuration/columns/#dataType'). For boolean columns, it is **true**; for other types of columns - **false**. 
```
